packages/x-components/src/design-system/components/picture/default.scss

```diff
@@ -1,48 +1,20 @@
 .x-picture {
-  // layout
-  display: block;
-  overflow: hidden;
```

Comment:
`overflow` is removed as will be inherited from x-tailwind, and `block` is removed as it's being applied in an already HTML block element (`div`)

Reply:
Be sure to set the `overflow: hidden` in the deprecated styles.

Reply:
I would move the `display: block` to the deprecated styles too, for the same reason as [here](https://github.com/empathyco/x/pull/1025#discussion_r1083994594).